### PR TITLE
fix(52761): overload tag is not supported in completions

### LINKED ISSUE #52761: `@overload` tag is not supported in completions
```ts
/**
 * @$
 */
```

Request completions at `$`.

Expected: `@overload`.
Actual: `@overload` is not included.

### PR DESCRIPTION
**Pull request by a-tarasyuk**
Fixes #52761

### CODE REVIEW

File: tests/cases/fourslash/jsdocOverloadTagCompletion.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+///<reference path="fourslash.ts" />
+
+// @noEmit: true
+// @allowJS: true
```
Comment (DanielRosenwasser) [inline tests/cases/fourslash/jsdocOverloadTagCompletion.ts:4]: We really don't need these directives.

**Review by DanielRosenwasser — APPROVED** [review]

File: tests/cases/fourslash/jsdocOverloadTagCompletion.ts (hunk 1 above)
Comment (DanielRosenwasser) [reply tests/cases/fourslash/jsdocOverloadTagCompletion.ts:4]: At least, we don't need `noEmit`.

**Comment by DanielRosenwasser** [comment]
You gotta let me know if you're planning on racing me with a PR :D

File: tests/cases/fourslash/jsdocOverloadTagCompletion.ts (hunk 1 above)
Comment (a-tarasyuk) [reply tests/cases/fourslash/jsdocOverloadTagCompletion.ts]: I’ve removed noEmit

**Comment by a-tarasyuk** [comment]
> You gotta let me know if you're planning on racing me with a PR :D

Sure 😄

**Pull request merged**